### PR TITLE
Fixed golint error

### PR DESCRIPTION
**Pull request by golisai**
Signed-off-by: Sri Goli <srgoli@cisco.com>

### CODE REVIEW

File: utils/sysutils.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type SystemAttributes struct {
 	TotalNetBw  int
 }
 
-// Exported system attributes
+// SysAttrs Exported system attributes
```
Comment (mapuri) [inline utils/sysutils.go]: @golisai thanks for the fix. Minor nit, may be we should reword this to:

```
SysAttrs are the exported system attributes
```

**Comment by jainvipin** [comment]
LGTM

File: utils/sysutils.go (hunk 1 above)
Comment (golisai) [inline utils/sysutils.go]: Sure. I will reword and resubmit.

**Comment by mapuri** [comment]
LGTM

cc/ @contiv/core-network for final review and merge
